jenkins-build.sh

```diff
@@ -35,7 +35,7 @@ cd $GOPATH/src/github.com/control-center/serviced
 # First, run the tests that require root
 #
 BUILD_TAGS="$(sudo bash ${DIR}/build-tags.sh) integration"
```

Comment:
It would be nice to add some echo statements before/after the sudos to explain that we're starting/finishing tests that require root
```
echo "Running tests that require root ..."
sudo su - root -c "source /home/jenkins/.gvm/scripts/gvm; gvm use go1.4.2; cd $PWD; GOPATH=$GOPATH godep go test -tags=\"${BUILD_TAGS}\" ./..."
echo "Finished tests that require root"
```